Formula/opencl-clhpp.rb

```diff
@@ -0,0 +1,36 @@
+class OpenclClhpp < Formula
```

Comment:
I think we need to have different name like `opencl-clhpp-headers` to be compatible with how OpenCL headers are named in Debian or other systems:
- `opencl-clhpp-headers` for C++
- `opencl-c-headers` for C